src/pages/projectSettings/tabs/EventLogTab/EventLogTab.tsx

```diff
@@ -38,11 +55,112 @@ export const EventLogTab: React.FC = () => {
     },
   });
 
+  const eventData: LogEntry[] = isRepo
+    ? repoEventData?.repoEvents?.eventLogEntries
+    : projectEventData?.projectEvents?.eventLogEntries || [];
+
+  return (
+    <div data-cy="event-log">
+      {eventData.map(({ user, timestamp, before, after }) => (
+        /* @ts-expect-error */
+        <EventLogCard key={`event_log_${timestamp}`}>
+          <EventLogHeader user={user} timestamp={timestamp} />
+          <Table
+            data={getEventDiffLines(before, after)}
+            columns={[
+              <TableHeader
+                key="key"
+                label="Property"
+                sortBy={(datum: EventDiffLine) => datum.key}
+              />,
+              <TableHeader
+                key="before"
+                label="Before"
+                sortBy={(datum: EventDiffLine) => datum.before}
+              />,
+              <TableHeader
+                key="after"
+                label="After"
+                sortBy={(datum: EventDiffLine) => datum.after}
+              />,
+            ]}
+          >
+            {({ datum }) => (
+              <Row key={datum.key} data-cy="event-log-table-row">
+                <Cell>
+                  <StyledCell>{datum.key ?? ""}</StyledCell>
+                </Cell>
+                <Cell>
+                  <StyledCell>{getEventValue(datum.before)}</StyledCell>
+                </Cell>
+                <Cell>
+                  <StyledCell>{getEventValue(datum.after)}</StyledCell>
+                </Cell>
+              </Row>
+            )}
+          </Table>
+        </EventLogCard>
+      ))}
+    </div>
+  );
+};
+
+interface Props {
+  timestamp: Date;
+  user: string;
+}
+
+const EventLogHeader: React.VFC<Props> = ({ user, timestamp }) => (
+  <StyledHeader>
+    <H3>{getDateCopy(timestamp)}</H3>
+    <div> {user} </div>
+  </StyledHeader>
+);
+
+/* @ts-expect-error */
+const EventLogCard = styled(Card)`
+  margin-bottom: ${size.l};
+  padding: ${size.m};
+  width: 150%;
+`;
+
+const StyledCell = styled("pre")`
+  word-break: break-all;
+  font-size: 12px;
+  font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+`;
+
+const StyledHeader = styled("div")`
+  padding-bottom: ${size.l};
+  padding-left: ${size.xxs};
+`;
+
+const getEventValue = (value: any) => {
+  if (value === null || value === undefined) {
+    return "";
+  }
+  if (typeof value === "boolean") {
+    return value ? "true" : "false";
+  }
+
+  if (value === "") {
+    return '""';
+  }
```

Comment:
[nit] This check isn't necessary because the next `if` statement will catch empty strings!